x-pack/plugins/ml/public/application/components/data_grid/data_grid.tsx

```diff
@@ -261,6 +263,16 @@ export const DataGrid: FC<Props> = memo(
       }
     }
 
+    const onMutation = () => {
+      const els = document.querySelectorAll('.euiDataGrid__virtualized');
+      // Our TypeScript lacks recognizing NodeList as iterable
+      for (const el of els as any) {
```

Comment:
have you tried `as unknown as NodeListOf<T>`?

Reply:
this didn't work unfortunately, found another fix with using `Array.from()`.